src/main/java/com/g4s8/ghman/data/EnvironmentVariables.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.data;
+
+/**
+ * Environment Variables Retriever.
+ *
+ * @since 1.0
+ * @checkstyle NonStaticMethodCheck (100 lines)
+ */
+public final class EnvironmentVariables {
+
+    /**
+     * Retrieve GitHub Client Id.
+     * @return Github Client Id
+     */
+    public String getGithubClientId() {
+        return System.getenv("GH_CLIENT");
```

Comment:
@samihibrahim correct this method to retrieve value from the field. If value is not found throw `IlligalStateException`

Reply:
@samihibrahim create unit test for this class, please

Reply:
@olenagerasimova fixed.